src/components/Nav.vue

```diff
@@ -1,68 +1,68 @@
+            <span>Dominican Open Source</span>
+          </div>
         </div>
+        <a href="#" data-target="mobile-nav" class="sidenav-trigger">
+          <i class="material-icons">menu</i>
+        </a>
+        <ul class="sections right hide-on-med-and-down">
+          <li :class="{ active: ['PopularRepositories', 'NewRepositories'].includes($route.name) }">
+            <router-link to="/repositories/popular">Repositories</router-link>
+          </li>
+          <li :class="{ active: ['PopularDevelopers', 'RecentlyJoinedDevelopers'].includes($route.name) }">
+            <router-link to="/developers/popular">Developers</router-link>
+          </li>
+          <li :class="{ active: $route.name == 'About' }">
+            <router-link to="/about">About</router-link>
+          </li>
+        </ul>
       </div>
-      <ul class="sections">
-        <li :class="{ active: ['PopularRepositories', 'NewRepositories'].includes($route.name) }">
-          <router-link to="/repositories/popular">Repositories</router-link>
-        </li>
-        <li :class="{ active: ['PopularDevelopers', 'RecentlyJoinedDevelopers'].includes($route.name) }">
-          <router-link to="/developers/popular">Developers</router-link>
-        </li>
-        <li :class="{ active: $route.name == 'About' }">
-          <router-link to="/about">About</router-link>
-        </li>
-      </ul>
     </nav>
+    <ul class="sections sidenav" id="mobile-nav">
+      <li :class="{ active: ['PopularRepositories', 'NewRepositories'].includes($route.name) }">
+        <router-link to="/repositories/popular">Repositories</router-link>
+      </li>
+      <li :class="{ active: ['PopularDevelopers', 'RecentlyJoinedDevelopers'].includes($route.name) }">
+        <router-link to="/developers/popular">Developers</router-link>
+      </li>
+      <li :class="{ active: $route.name == 'About' }">
+        <router-link to="/about">About</router-link>
+      </li>
+    </ul>
   </div>
 </template>
 
 <script>
 export default {
-  name: "Nav"
+  name: "Nav",
+  mounted() {
+    M.AutoInit();
+  }
 };
 </script>
 
 <style lang="scss" scoped>
-.header-wrapper {
-  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
-  .header {
-    display: flex;
-    justify-content: space-between;
-    box-shadow: none;
-    align-items: center;
-
-    .brand-logo {
-      position: initial;
-      color: #fff;
-      font-size: 1.9rem;
-      svg {
-        fill: currentcolor;
-        width: 1.5em;
-        vertical-align: middle;
-        margin-right: 8px;
-        margin-top: -5px;
-      }
-      span {
-        font-size: 2.1rem;
-      }
-    }
-
-    ul {
-      display: flex;
-      li {
-        font-size: 1em;
-      }
-    }
+.brand-logo {
+  color: #fff;
+  font-size: 1.9rem;
+  svg {
+    fill: currentcolor;
+    width: 1.5em;
+    margin-right: 8px;
+    margin-top: 10px;
+  }
+  span {
+    font-size: 2.1rem;
```

Comment:
Add a media query to change the font size to `1.6rem` when the window width is <= 768px (tablet size)

Reply:
Done in the last commit.